### ISSUE TITLE
[README] Programmatic Integrations options are missing some CLI options

### ISSUE THREAD

**Issue by rob-ryszewski** [pinojs/pino-pretty#348]
Getting all the CLI options and API options from the README:
```bash
# omitting options that are truly API specific 
# ie: sync, mkdir, append, destination
$ cat README.md | 
    grep -A1 -B27 -e 'customPrettifiers: {}' | 
    cut -d: -f1 | 
    grep -v -e // -e '[{}]' -e '^ *$' | 
    cut -d' ' -f3 | 
    grep -v -e '^sync$' -e '^mkdir$' -e '^append$' -e '^destination$' | 
    sort > api.args

$ cat README.md | 
    grep -e '`-\-' |
    cut -d'`' -f2 | 
    cut -d'-' -f3 | 
    sort > cli.args 

$ git diff --no-index cli.args api.args                                                                                                                                                                          
```
```diff
diff --git a/cli.args b/api.args
index eb025bc..53c3bbe 100644
--- a/cli.args
+++ b/api.args
@@ -1,19 +1,14 @@
 colorize
-config
 crlf
-customColors
-customLevels
+customPrettifiers
 errorLikeObjectKeys
 errorProps
 hideObject
 ignore
 levelFirst
 levelKey
-levelLabel
 messageFormat
 messageKey
-minimumLevel
 singleLine
 timestampKey
 translateTime
-useOnlyCustomProps
```

In short, the following options are missing from the Programatic Integration section of the README:
- `config`
- `customColors`
- `customLevels`
- `levelLabel`
- `minimumLevel`
- `useOnlyCustomProps`

Just a heads up, fairly minor issue.

**Comment by mcollina** [comment]
Would you like to send a Pull Request to address this issue?

**Comment by or2008** [comment]
+1

**Comment by ademarsj** [comment]
Isn't that closed by the PR #381 ? It's already merged on Oct 9, 2022.
